src/common/constants.ts

```diff
@@ -120,6 +121,8 @@ export class Constants {
     public static moduleNameSubstitution = "${moduleName}";
     public static repositoryNameSubstitution = "${repositoryName}";
     public static selectDefaultPlatform = "Select Azure IoT Edge Solution Default Platform";
+    public static sampleName = "Sample Name";
+    public static sampleNamePrompt = "Provide a Sample Name";

```

Comment:
Minor wording comment: "Provide a Sample Name" might be confusing for users because sample name like "custom vision" should not be able to be configured by users. I prefer something like "Provide a Solution Name for the Sample". I'll leave @shizn to decide.

Reply:
Good suggestion, I think you are right!